src/navigation/ackermann_motion_primitives.cc

```diff
@@ -61,21 +61,21 @@ void AckermannSampler::SetMaxPathLength(ConstantCurvatureArc* path_ptr) {
     return;
   } 
   const float turn_radius = 1.0f / path.curvature;
-  const Vector2f turn_center(0, turn_radius);
-  const Vector2f target_radial = local_target - turn_center;
-  const Vector2f middle_radial =
-      fabs(turn_radius) * target_radial.normalized();
-  const float middle_angle =
-      atan2(fabs(middle_radial.x()), fabs(middle_radial.y()));
-  const float dist_closest_to_goal = middle_angle * fabs(turn_radius);
+  // const Vector2f turn_center(0, turn_radius);
```

Comment:
Are these variables unused? If so, this commented out block can be deleted.

Reply:
sure if you're ok with me deleting this whole `closest_to_goal` truncation I will definitely delete the code